docs/reference/settings/data-frames-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+
+[role="xpack"]
+[[data-frames-settings]]
+=== Data frames settings in Elasticsearch
```

Comment:
Unless other types of data frame settings will be added here, I suggest changing it to:
```suggestion
=== {dataframe-transforms-cap}  settings in Elasticsearch
```